model/system/metrics.yaml

```diff
@@ -1,4 +1,17 @@
 groups:
+  # system.* metrics
+  - id: metric.system.uptime
+    type: metric
+    metric_name: system.uptime
+    stability: experimental
+    brief: "The time the system has been running"
+    note: |
+      Instrumentations SHOULD use counter with type `double` and measure uptime with the highest precision available.
+      The actual accuracy would depend on the instrumentation and operating system.
+    instrument: counter
```

Comment:
it seems @jsuereth has good argument to make it a gauge in 
https://github.com/open-telemetry/oteps/blob/05ff3aed8d6d9b9625e4756b4c279048d46e3b28/text/metrics/0185-uptime-monitoring-semantic-conventions.md#uptime

Reply:
Yeah based on that OTEP, I think this should be changed to `gauge`. I'm going to take that OTEP and work with @jsuereth and the other folks who originally authored the guidance to turn this into official System Semconv guidance. So while it's not yet, we'll soon attempt to make it a proper rule that uptime metrics should be a gauge, and will add references to that guidance in the note for this and any future uptime metrics once it's ready.

Reply:
updated